### ISSUE TITLE
Is it possible to add  user-defined overlay layers？

### ISSUE THREAD

**Issue by anders86755** [mapbox/geojson.io#854]
Is it possible to add  user-defined overlay layers that can be superimposed on top of the base map layer, with adjustable transparency?
I made this modification using an old version. I was wondering if you could add this feature to the new version and make it an official function? It would be very valuable.
just like this：
![微信截图_20230614092621a](https://github.com/mapbox/geojson.io/assets/40052500/b67ccb5e-baf1-42ae-afe7-6d56ffe2c085)
![微信截图_20230614092621](https://github.com/mapbox/geojson.io/assets/40052500/7250a69d-41fc-4dfb-ad19-12ad8d9f6c18)

**Comment by chriswhong** [comment]
How would you use this feature?  Would you just want to have a reference layer on the map when editing geojson?  I think this is similar to allowing users to bring their own raster tileset, which geojson.io already does.

A workaround in the meantime could be to just add whatever features you would want in this overlay and use simplestyle to style them.

**Comment by anders86755** [comment]
Yes, there is indeed a similar functionality available in GeoJSON. However, the provided overlay is in the format of XYZ.png, while there is another format that also very common, which is the WMS (Web Map Service) approach. It would be perfect if official release could include WMS.  I am using WMS.
Thank you for your response.

**Comment by chriswhong** [comment]
Closed by #1009
